README.md

```diff
@@ -164,69 +143,46 @@ npm run test
-│   ├── constants/                 # Names, addresses, static configurations and other values used throughout the application
-│   ├── context/                   # React context configuration files
-│   ├── hooks/                     # React custom hooks
-│   ├── store/                     # Redux store resides here.
-│   │   ├── actions/               # Store actions reside here and are broken into script files dedicated to each system entity.
-│   │   ├── middlewares/           # All the Redux middlewares are places here and have their dedicated script files based on the system entities.
-│   │   ├── reducers/              # Redux reducers are located here. similar to actions and reducers, they are placed in script files named after the entity they represent.
-│   ├── utils/                     # Utility functions
-└──test/                           # E2E tests written with Cypress.io and Cucumber; also some helpers used by unit test that live in /src
+│   ├── const/                     # Static configurations and other values used throughout the application.
+│   ├── locales/                   # Contains the built localization files.
+│   ├── modules/                   # Divided logic in domain specific areas, an example structure is shown below.
+│   │   ├── hardwareWallet/        # Module/Domain.
+│   │       ├── __fixtures__/      # Contains mock data for the hardwareWallet module.
+│   │       ├── components/        # Components related to the hardwareWallet domain.
+│   │       ├── hooks/             # Hooks related to the hardwareWallet domain.
+│   │       ├── store/             # Redux encapsulated logic related to the hardwareWallet domain.
+│   │           ├── actions/       # HardwareWallet actions.
+│   │           ├── selectors/     # HardwareWallet selectors.
+│   │           ├── reducers/      # HardwareWallet reducers.
+│   │       ├── utils/             # Utils used in the hardwareWallet module. Global utils should be put in src/utils.
+│   ├── redux/                     # Contains the root reducer, all other reducers are imported here. It also has legacy pattern, selectors, and actions (these should now be encapsulated in its specific module).
+│   ├── routes/                    # Contains all routes in the application as well as modals.
+│   │   ├── routes.js              # Route and modal paths with metadata. This object is used in all our links.
+│   │   ├── routesMap.js           # Maps components to routes, this is used in the MainRouter.js.
+│   ├── service/                   # Add services, such as mock services.
+│   ├── theme/                     # Themed components (Button, Input, ...).
+│   ├── utils/                     # Global utility functions used throughout the app.
+        ├── i18n/                  # Localization setup.
+└──test/                           # E2E tests written with Cypress.io and Cucumber; also some helpers used by unit test that live in /src.
```

Comment:
Imho, the line 185 should read as follows:
E2E tests written with Cypress.io and Cucumber; also some helpers used by unit _tests_ that live in /src.